packages/uploadthing/package.json

```diff
@@ -20,6 +20,10 @@
     "./next-legacy": {
       "import": "./dist/next-legacy.mjs",
       "types": "./dist/next-legacy.d.ts"
+    },
+    "./nuxt": {
+      "import": "./dist/nuxt.mjs",
+      "types": "./dist/nuxt.d.ts"
```

Comment:
you'll also need to set the `typesVersion`

Reply:
It is already set there. Are you referring something else?
<img width="584" alt="image" src="https://github.com/pingdotgg/uploadthing/assets/55754540/d812592d-b6c9-47e2-84dc-7b84ea720842">

Reply:
i think you might have to update your branch then cause that's not up to date:

https://github.com/pingdotgg/uploadthing/blob/9534d60efb0767de57ddd8b242400623ccceafc0/packages/uploadthing/package.json#L28-L43